hyperjump

```diff
@@ -149,19 +148,16 @@ function jj() {
 		local param
 		for param in ${@:2}
 		do
-			if [ ! -z "$param" -a "$param" != "" ]
-			then
+			if [[ ! -z "$param" -a "$param" != "" ]]; then
```

Comment:
If you are going to change to [[, you also need to change -a to &&. Did you try running the changes?